tsconfig.json

```diff
@@ -28,26 +36,65 @@
       }
     ],
     "paths": {
-      "@payload-config": ["./test/admin/config.ts"],
-      "@payloadcms/live-preview": ["./packages/live-preview/src"],
-      "@payloadcms/live-preview-react": ["./packages/live-preview-react/src/index.ts"],
-      "@payloadcms/ui/assets": ["./packages/ui/src/assets/index.ts"],
-      "@payloadcms/ui/elements/*": ["./packages/ui/src/elements/*/index.tsx"],
-      "@payloadcms/ui/fields/*": ["./packages/ui/src/fields/*/index.tsx"],
-      "@payloadcms/ui/forms/*": ["./packages/ui/src/forms/*/index.tsx"],
-      "@payloadcms/ui/graphics/*": ["./packages/ui/src/graphics/*/index.tsx"],
-      "@payloadcms/ui/hooks/*": ["./packages/ui/src/hooks/*.ts"],
-      "@payloadcms/ui/icons/*": ["./packages/ui/src/icons/*/index.tsx"],
-      "@payloadcms/ui/providers/*": ["./packages/ui/src/providers/*/index.tsx"],
-      "@payloadcms/ui/templates/*": ["./packages/ui/src/templates/*/index.tsx"],
-      "@payloadcms/ui/utilities/*": ["./packages/ui/src/utilities/*.ts"],
-      "@payloadcms/ui/scss": ["./packages/ui/src/scss.scss"],
-      "@payloadcms/ui/scss/app.scss": ["./packages/ui/src/scss/app.scss"],
-      "@payloadcms/next/*": ["./packages/next/src/*"],
-      "@payloadcms/next": ["./packages/next/src/exports/*"]
+      "@payload-config": [
+        "./test/fields/config.ts"
```

Comment:
@jacobsfletch I think we should keep this as `"./test/admin/config.ts"`, no?

Reply:
This value dynamically changes back and forth based on the test suite that is identified by the script, i.e. `pnpm dev fields` injects `fields` into the tsonfig on the fly. So no need to ignore this change. It will simply change back when the next dev spins up another suite.